packages/kmod-6.1-neuron/neuron-systemd-modules-load.drop-in.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+ExecStartPre=-/usr/bin/depmod
```

Comment:
This won't be necessary after we update to a `twoliter` release with this change:
https://github.com/bottlerocket-os/twoliter/pull/205